src/lib.rs

```diff
@@ -359,14 +429,15 @@ impl LocalSizeClass {
         );
 
         let mut mmap = Self::init_file(byte_len)?;
-        // SAFETY: Changing the lifetime of the mapping to static.
-        let area: Mem = unsafe { std::mem::transmute(&mut *mmap) };
-        let mut chunks = area.chunks_mut(self.size_class.byte_size());
-        let mem = chunks.next().expect("At least once chunk allocated.");
+        let mut chunks = mmap.as_mut().chunks_mut(self.size_class.byte_size());
+        let mem = chunks
+            .next()
+            .expect("At least once chunk allocated.")
+            .into();
```

Comment:
I think we should use an unsafe function instead of `From`, as re-dereferencing the `mmap` anytime after this is possible UB (actually, im not sure if having a mutable pointer alias a `&mut` is ub (i _think_ its not), but definitely accessing a second mutable reference or pointer is